### DISCUSSION TITLE
Consider implementing conditional duplication tracking

### DISCUSSION THREAD

**Discussion by mmikkel** [verbb/super-table#478] [Ideas]
I came across this when putting together #477. Craft [recently](https://github.com/craftcms/cms/commit/b755f388bbd0563209e1d9f624c61321bd9c870d) implemented the concept of conditional duplication tracking, fixing https://github.com/craftcms/cms/issues/10906 which could make Matrix blocks disappear (in what is arguably an edge-case scenario, but still). 

I haven't attempted to replicate the issue with SuperTable blocks, but assuming the behaviour described would affect SuperTable as well, it might be worth implementing the same fix?

**Comment by engram-design** [comment]
Ah yes, I believe that indeed would be an issue. I'll check it out!
